app/components/views/GetStartedPage/PreCreateWallet/hooks.js

```diff
@@ -115,6 +130,10 @@ export const usePreCreateWallet = ({
       trezorAlertNoConnectedDevice();
       return;
     }
+    if (isLedger && !ledgerDevice) {
+      ledgerAlertNoConnectedDevice();
+      return;
+    }
```

Comment:
```suggestion
    if (isLedger && !ledgerDevice) {
      return ledgerAlertNoConnectedDevice();
    }
```

Reply:
Changed but does this return value do anything?